deployment/terraform/dataspace/main.tf

```
@@ -151,11 +153,18 @@ resource "azurerm_storage_blob" "dataspace_did" {
   storage_container_name = "$web" # container used to serve static files (see static_website property on storage account)
   type                   = "Block"
   source_content = jsonencode({
-    id = local.dataspace_did_url
+    id = local.dataspace_did_uri
     "@context" = [
       "https://www.w3.org/ns/did/v1",
       {
-        "@base" = local.dataspace_did_url
+        "@base" = local.dataspace_did_uri
+      }
+    ],
+    "service" : [
+      {
+        "id" : "#enrollment-url",
+        "type" : "EnrollmentUrl",
```

nit: maybe we can say registration-service-url as this is the base URL for registration service.
```suggestion
        "id" : "#registration-service-url",
        "type" : "RegistrationServiceUrl",
```

---

what about `RegistrationUrl`?

---

Changed to RegistrationUrl after discussing offline